OxfordBodleian/Juel-Jensen/BDLaethe36.xml

```diff
@@ -161,8 +161,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     The numeral <foreign xml:lang="gez">፮</foreign> has a compressed shape and the open loop.
                                 </desc>                               
                                 <seg type="rubrication">
-                                    Rubricated are the number and titles of the psalms, the stichometry, indicated by the word <foreign xml:lang="gez">ቃሉ፡</foreign> 
-                                    followed by a numeral, and elements of the numerals and of the punctuation signs, including text dividers.
+                                    The number and titles of the psalms; the stichometry, indicated by the word <foreign xml:lang="gez">ቃሉ፡</foreign> 
```

Comment:
Why here "the" number...
without in BDLaethe30 and  BDLaethe33

Reply:
Should say. Rubrication: "the number and title of the psalms" or "the numbers and titles of the psalms" for consistency. 

As for the "the" before the list, we can choose to include or exclude it as long as its consistent, I think we had opted for a more schematic approach.

Reply:
Changed into "numbers and titles of the psalms"